cloudflare.go

```diff
@@ -464,13 +481,41 @@ func (worker *CloudflareWorker) UpdateIPLists() error {
 		}
 	}
 
-	for action, state := range newIPSetByAction {
-		if reflect.DeepEqual(worker.CFStateByAction[action].IPListState.IPSet, state) || len(state) == 0 {
+	for action, set := range newIPSetByAction {
+		if reflect.DeepEqual(worker.CFStateByAction[action].IPListState.IPSet, set) {
 			log.Info("no changes to IP rules ")
 			continue
 		}
+		if len(set) == 0 {
```

Comment:
further logging warning that we're emptying list might be good, it's exceptional enough :)

Reply:
Fixed via https://github.com/crowdsecurity/cs-cloudflare-bouncer/pull/56/commits/8b56fccf8f992071713099b8d387df061530fcee